### PR TITLE
commit classwork 16/03/2024 oop practice

### PR DESCRIPTION
**Pull request by Dilnaza07**
oop practice tasks

### CODE REVIEW

File: src/CodingBat/Warmup_1/frontBackTask_11.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package CodingBat.Warmup_1;
+
+public class frontBackTask_11 {
+    public static void main(String[] args) {
+
+    }
+
+   // public static String frontBack(String str) {
```
Comment (CoolFire075) [inline src/CodingBat/Warmup_1/frontBackTask_11.java]: remove comments

Comment (CoolFire075) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: what are you doing?

Comment (CoolFire075) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: j47rgrwiufyg

Comment (CoolFire075) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: rtwgwt4g

Comment (CoolFire075) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: 4w5h4w5h

**Review by SuperPuperPolyfabricatV40000 — COMMENTED** [review]
HAHHAAA
I AM HATER

File: src/CodingBat/Warmup_1/frontBackTask_11.java (hunk 1 above)
Comment (Dilnaza07) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: sleep

Comment (SuperPuperPolyfabricatV40000) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: sdjf;klsdjl;fkjsd;fjsd;f

Comment (SuperPuperPolyfabricatV40000) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: png-clipart-spongebob-squarepants-creature-from-the-krusty-krab-wii-patrick-star-spongebob-s-atlantis-squarepantis-mr-krabs-others-miscellaneous-game

**Review by SuperPuperPolyfabricatV40000 — COMMENTED** [review]
I love sponges__

**Comment by CoolFire075** [comment]
shut up hater!

File: src/CodingBat/Warmup_1/frontBackTask_11.java (hunk 1 above)
Comment (SuperPuperPolyfabricatV40000) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: Gumburgers

Comment (CoolFire075) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: who is he?

Comment (CoolFire075) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: don't trouble me

Comment (SuperPuperPolyfabricatV40000) [reply src/CodingBat/Warmup_1/frontBackTask_11.java:8]: png-clipart-spongebob-squarepants-creature-from-the-krusty-krab-wii-patrick-star-spongebob-s-atlantis-squarepantis-mr-krabs-others-miscellaneous-game

Comment (SuperPuperPolyfabricatV40000) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: SPONGEBOB ATTACK MY ENEMIES!!!!
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Comment (SuperPuperPolyfabricatV40000) [reply src/CodingBat/Warmup_1/frontBackTask_11.java]: Food it's very delicious Spongbob

**Pull request merged**